cyclonedx-lib/src/temurin/sbom/TemurinGenSBOM.java

```diff
@@ -111,6 +113,11 @@ public static void main(final String[] args) {
                 writeJSONfile(bom, fileName);
                 break;
 
+            case "signSBOM":
+                bom = signBom();
+                new JSONObjectWriter().setSignature(new JSONAsymKeySigner(privateKey, publicKey, null));
```

Comment:
this was just a snippet. not a real working code.
does it work for you locally, when you compile it?